samples/cpp/dbt_face_detection.cpp

```diff
@@ -1,4 +1,4 @@
-#if defined(__linux__) || defined(LINUX) || defined(__APPLE__) || defined(ANDROID)
+#if defined(__linux__) || defined(LINUX) || defined(__APPLE__) || defined(ANDROID) || (defined(_MSC_VER) && _MSC_VER>1700)
```

Comment:
actually, >1700 was right, unless there is a release between VS2012 and VS2013. VS2013 is 1800. So now it's clearer.